.github/workflows/build.yml

```diff
@@ -129,9 +165,24 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      - name: Login to Quay
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
```

Comment:
have we setup these env?

Reply:
Yes @kmova Can we set up these envs before merging this PR.

Reply:
@pawanpraka1 @shubham14bajpai -- updated the ENV